Sources/SwiftSyntaxBuilder/Syntax+StringInterpolation.swift

```diff
@@ -108,19 +108,35 @@ extension SyntaxStringInterpolation: StringInterpolationProtocol {
   }
 
   // Append a value of any CustomStringConvertible type as source text.
+  @available(*, deprecated, renamed: "appendInterpolation(raw:)", message: "use '\\(raw: <value>)' to interpolate a plain string directly into Swift code")
```

Comment:
This is a new API anyway, so I'd say it's fine if you delete it. But also okay if you want to leave it in. It's possible that swift-format/lint have already updated to use it.

Reply:
The deprecation exists both to correct existing code, and to tell people what to do if they're writing brand-new code and naïvely use a plain interpolation. Granted, it should probably mention `\(literal:)` now that that's an option.